docs/source/onboarding_resources/computing/index.rst

```diff
@@ -125,39 +125,88 @@ in the Accessing the Cluster section below.
+on the command line in Module 1 of the IHME Learn training `Computational Infrastructure Level 1 <https://ihme.brightspace.com/d2l/home/7028>`_. Command lines technically interact with an 
+operating system on your computer. 
 
+Operating systems are things like Windows or Mac that help a computer run and provide some basic 
+infrastructure. Opening and saving files all happen through your operating system. You can change or 
+install multiple operating systems, but that is quite uncommon. 
 
-**How to Access GitHub:** Similar to the cluster, Git commands are written into a 
-terminal to access the repositories and push edits. On our team, most Windows folks use 
-Git Bash which automatically installs with downloading Git. However, some prefer 
-the Windows Subsystem for Linux (WSL) for it's user interface and tools. 
+Some operating systems require different command lines to accomplish the same task. The IHME 
+cluster uses the operating system Linux, and so the trainings provided are designed for Linux. 
+Linux and Mac are very similar, but Windows is quite different. Therefore, we recommend for 
+Windows users to install a terminal that can accept command lines written for a Linux while 
+still allowing you to still "speak" with your Windows computer. 
 
-For Mac users, the Terminal app which comes installed on your machine can be used for 
-Git as well. 
+**Recommended Terminals for Windows Users:** 
 
-It is worth noting here that these applications can also be used to access the cluster, 
-although the common practice on the Sim Science team is to use separate terminals for 
-GitHub on our local machine and for cluster access. 
+For updating Vivarium docs and interacting with GitHub, we recommend using Git Bash. This 
+installs automatically with Git for Windows. However, some prefer 
+the Windows Subsystem for Linux (WSL) for its user interface and tools. 
 
 Link to download `Git and Git Bash <https://git-scm.com/>`_
 
 Link to download `WSL <https://learn.microsoft.com/en-us/windows/wsl/install>`_
 
-One difference in uploading to GitHub is that the cluster will require a password 
-to push information. This is **NOT** your GitHub password, but instead is a unique 
-token that you will need to create. `This website <https://techglimpse.com/git-push-github-token-based-passwordless/>`_ has information on creating a token. Many Sim Science users set their token to 
+For accessing the cluster, we recommend using PuTTy or Bitvise. How to access the cluster 
+is included in more depth in Section 3. 
+
+**Recommended Terminals for Mac Users:** 
+
+Since Mac is similar to Linux and is the base operating system on Mac computers, the 
+pre-installed Terminal app can be used for all your terminal needs. 
+
+**Other Options:**
+
+Most terminals can also be used to access the cluster, although the common practice on 
+the Sim Science team is to use separate terminals for GitHub on our local machine and 
```

Comment:
Should "GitHub" here be "Git Bash"?
```suggestion
the Sim Science team is to use separate terminals for Git Bash on our local machine and 
```

Reply:
I think the meaning intended here is "to work with GitHub on our local machine," while Git Bash _is_ the terminal used to do this.

```suggestion
the Sim Science team is to use separate terminals for working on our local machine and 
```

Reply:
Yes - thanks I did clarify this to be working on your local machine.